### PR TITLE
SLING-10911 Update to Parent 46

### PR DESCRIPTION
**Pull request by enapps-enorman**
Update to the latest parent so the module can be built with java 11

Also migration to the standard scr annotations

### CODE REVIEW

File: bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Export-Package:\
```
Comment (kwin) [inline bnd.bnd:1]: Every versioned package is automatically exported

File: bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  org.apache.sling.launchpad.testservices.exported,\
+  org.apache.sling.launchpad.testservices.models,\
+  org.apache.sling.launchpad.testservices.events
+
+Private-Package:\
```
Comment (kwin) [inline bnd.bnd:6]: No need for this non-standard header

File: bnd.bnd (hunk 1 above)
Comment (enapps-enorman) [reply bnd.bnd]: Thanks for the review.  This was a direct copy of how the old maven-bundle-plugin instructions were doing it.  And yes, you are correct, but only one of those exported packages contained a package-info.java file so a little more work is needed here.

I have cleaned this up by adding a package-info.java file to the other two packages and removed this instruction.

File: bnd.bnd (hunk 2 above)
Comment (enapps-enorman) [reply bnd.bnd]: Thanks for the review.  This was a direct copy of how the old maven-bundle-plugin instructions were doing it.   I don't have a problem removing this instruction as it doesn't appear to have any impact on running the tests.

However, I would note that removing this instruction doesn't prevent this attribute from showing up in the generated manifest as bnd would just use the default configuration.

I've seen a few places using the "removeheaders" instruction to remove it from the output, is this what you were expecting?

-removeheaders:\
  Private-Package

Comment (kwin) [reply bnd.bnd]: No need for removing, just go with the standard value for `Private-Package` as generated by bnd.

Comment (enapps-enorman) [reply bnd.bnd]: fair enough.  This was already done with the last commit.

**Review by kwin — APPROVED** [review]
LGTM, thanks

**Pull request merged**